bpm/bonita-core/bonita-form-mapping/bonita-form-mapping-impl/src/main/java/org/bonitasoft/engine/core/form/impl/FormMappingServiceImpl.java

```diff
@@ -13,7 +13,13 @@
  */
 package org.bonitasoft.engine.core.form.impl;
 
-import static org.bonitasoft.engine.page.AuthorizationRuleConstants.*;
+import static org.bonitasoft.engine.page.AuthorizationRuleConstants.IS_ACTOR_INITIATOR;
```

Comment:
my import preference do not keep stars. I think its better this way. Is there something set in the eclipse code formatter that define how we handle that?